cliquet/tests/testapp/__init__.py

```diff
@@ -14,4 +14,6 @@ def main(settings=None, config=None, *args, **additional_settings):
         config = Configurator(settings=settings)
     cliquet.initialize(config, version='0.0.1')
     config.include(includeme)
-    return config.make_wsgi_app()
+    app = config.make_wsgi_app()
+    # Install middleware (idempotent if disabled)
```

Comment:
It means that if install_middleware actually doesn't install any middleware it will just return the app.